resources/provider/migrations/postgres/30_v0.11.4.down.sql

```diff
@@ -0,0 +1,7 @@
+-- Resource: rds.instances
+ALTER TABLE IF EXISTS aws_rds_instances DROP CONSTRAINT aws_rds_instances_pk;
+ALTER TABLE IF EXISTS aws_rds_instances ADD CONSTRAINT aws_rds_instances_pk PRIMARY KEY (account_id,id);
+
+-- Resource: ssm.instances
+ALTER TABLE IF EXISTS aws_ssm_instance_compliance_items DROP CONSTRAINT aws_ssm_instance_compliance_items_pk;
+ALTER TABLE IF EXISTS aws_ssm_instance_compliance_items ADD CONSTRAINT aws_ssm_instance_compliance_items_pk PRIMARY KEY (instance_cq_id,resource_id,id);
```

Comment:
would do truncations before and after, although I doubt arn will be nil

Reply:
Done.